src/d3a/models/area/__init__.py

```diff
@@ -59,10 +59,6 @@
 
 class Area:
 
-    reconfig_parameters = ('grid_fee_percentage', 'transfer_fee_const',
-                           'baseline_peak_energy_import_kWh', 'baseline_peak_energy_export_kWh',
-                           'import_capacity_kVA', 'export_capacity_kVA')
```

Comment:
Why are you completely getting rid of it?

Reply:
It is not used any more. This is a remnant of my first implementation of liveEvents.